index.js

```diff
@@ -25,7 +25,11 @@ module.exports = async robot => {
 
       // Some payloads don't include labels
       if (!issue.labels) {
-        issue = (await context.github.issues.get(context.issue())).data
+        try {
+          issue = (await context.github.issues.get(context.issue())).data
+        } catch (error) {
+          console.log('Issue not found')
```

Comment:
Can you use `context.log` here? That will give it some special formatting, and will also include information about the request in the log.

Reply:
Sure, it's done.